Update product designer exercise public description (#3271)
sourcegraph/about: handbook/product/roles/interviews/product_designer/pd_design_exercise.md

    * Understand the business requirements
    * Explore the scope of the project 
    * Determine a project plan for the design aspects of the project
    * Explore solutions for selecting repositories in a fidelity and design tool of the candidate’s choice. 

#### Please note: 

The chosen problem has been implemented, so interviewers will need to role-play their involvement slightly, but our goal is to make the exercise as close to a collaborative working session as possible. This does not mean our solution is the right or best solution, and we are not evaluating the outcomes of this session against what we’ve done in the past.

We are not providing the interview ahead of time to ensure all candidates have equal time to devote to the problem, regardless of external commitments. This means the session will be contrary to our typically async method of working, but we feel it is more important to respect the candidate’s time than to replicate our process exactly. 

#### We will evaluate the following:

* Can the candidate lead the room, keep the group focused, and facilitate the exercise.

> * What's the candidate process of exploring new problems and getting all the information necessary to complete the task

Author: ```suggestion
* How does the candidate explore new problems and get the information necessary to complete a task
```
